src/app.js

```diff
@@ -135,44 +134,46 @@ const App = Marionette.Application.extend({
 
     this.triggerMethod('before:start', options);
 
+    const opts = _.extend({}, options);
+    opts.state = this.getInitState(opts.state);
+
+    this.initState(opts);
+
     this._isRunning = true;
 
-    this.triggerStart(options);
+    this.triggerStart(opts);
 
     return this;
   },
 
   /**
-   * Triggers start event.
-   * Override to introduce async start
+   * Extends passed in state.
```

Comment:
_Sigh_ @paulfalgout I just realized that the screenshot was the same location as your comment...I thought it was in the docs...but I just realized that I need to add docs on this function.